pkg/handler/repository_rpm.go

```diff
@@ -74,16 +76,17 @@ func (rh *RepositoryRpmHandler) searchRpmPreprocessInput(input *api.SearchRpmReq
 // @Router       /repositories/{uuid}/rpms [get]
```

Comment:
need to add @Param options here similar to the search on repository list

Reply:
roger! 🖖

Reply:
add this here and to repository list... where I may have neglected that in the past. 🙄